v1/feed.js

```diff
@@ -133,8 +133,77 @@ Feed.prototype.aggregated = function(hyper, req) {
                 return finalResult;
             });
         });
-    });
+    })
+    .then(function(res) {
+        // We've got the titles, populate them with summaries
+        var feed = res.body;
+        var summaries = {};
+
+        function requestTitle(title) {
+            if (title && !summaries[title]) {
+                summaries[title] = hyper.get({
+                    uri: new URI([rp.domain, 'v1', 'page', 'summary', title])
+                })
+                .get('body')
+                // Swallow the error, no need to fail the whole feed
+                // request because of one failed summary fetch
+                .catchReturn(undefined);
+            }
+        }
+
+        if (feed.tfa) {
+            requestTitle(feed.tfa.title);
+        }
+        if (feed.mostread && feed.mostread.articles) {
+            feed.mostread.articles.forEach(function(article) { requestTitle(article.title); });
+        }
+        if (feed.random && feed.random.items) {
+            feed.random.items.forEach(function(article) { requestTitle(article.title); });
+        }
+        if (feed.news) {
+            feed.news.forEach(function(story) {
+                if (story.links) {
+                    story.links.forEach(function(article) {
+                        requestTitle(article);
+                    });
+                }
+            });
+        }
+
+        return P.props(summaries)
+        .then(function(summaries) {
+            function assignSummary(article) {
+                if (summaries[article.title]) {
+                    // MCS expects the title to be a DB Key
+                    delete summaries[article.title].title;
```

Comment:
Ah, good point. 

Too bad about the inconsistency between title properties in summary & feed end points. But, probably too late since the app already depends on it.